Geometry/MTDCommonData/test/runTest.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh -e
+
+function die { echo $1: status $2 ; exit $2; }
+function checkDiff {
+    FSIZE=$(stat -c%s "$1")
+    echo "The output diff is $FSIZE:"
+    cat $1;
+    if [ $FSIZE -gt 7455 ]
```

Comment:
Do you want to give some leeway in case there is a small increase in differences? Maybe use 7500 instead of 7455.
Also, please put a comment explaining why 7455 differences are OK.